extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -969,6 +959,12 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       setStyles(this.element, {width: `${size.width}px`});
     }
 
+    if (this.friendlyIframeEmbed) {
+      this.friendlyIframeEmbed.whenWindowLoaded().then(() => {
```

Comment:
Rather than exposing the friendly iframe embed object (allowing for other modifications), let's create a new function in AmpA4a = onFriendlyIframeEmbedLoaded

Reply:
Done.